repo/packages/C/confluent-control-center/3/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "3.0",
+  "name": "confluent-control-center",
+  "version": "0.9.6-3.1.1",
+  "scm": "https://github.com/confluentinc/control-center",
+  "description": "Confluent Control Center service",
```

Comment:
Done (temporary pointer to Confluent docs, until dcos/examples docs can be written)